types/index.d.ts

```diff
@@ -5205,6 +5206,13 @@ export interface CookiesNsCommands<ReturnType = unknown> {
   ): Awaitable<IfUnknown<ReturnType, this>, null>;
 }
 
+export interface FirefoxNsCommands<ReturnType = unknown> {
+  getContext(): Awaitable<IfUnknown<ReturnType, this>, 'content' | 'chrome'>;
+  setContext(ctx: 'content' | 'chrome' | PromiseLike<'content' | 'chrome'>): Awaitable<IfUnknown<ReturnType, this>, null>;
+  installAddon(path:string, temporary?: boolean): Awaitable<IfUnknown<ReturnType, this>, string>;
+  uninstallAddon(addonId: string | PromiseLike<string>): Awaitable<IfUnknown<ReturnType, this>, null>;
```

Comment:
Any specific reason to use `PromiseLike`?

Reply:
With `Promise` could not directly assign the result of `getContext()` as a input parameter of `setContext()`. Thus, instead of `Promise` used `PromiseLike`.